src/black/mode.py

```diff
@@ -207,12 +218,15 @@ def __contains__(self, feature: Preview) -> bool:
         """
         Provide `Preview.FEATURE in Mode` syntax that mirrors the ``preview`` flag.
 
-        The argument is not checked and features are not differentiated.
-        They only exist to make development easier by clarifying intent.
+        In unstable mode, all features are enabled. In preview mode, all features
+        except those in UNSTABLE_FEATURES are enabled. For legacy reasons, the
+        string_processing feature has its own flag, which is deprecated.
```

Comment:
Question- what's the policy around deprecated flags? When is `--experimental-string-processing` expected to be removed?

Reply:
There isn't a real policy. That flag has been deprecated since 2022 so I think it's about time it goes away.